src/components/Wallet/FeatureCards/index.tsx

```diff
@@ -0,0 +1,109 @@
+import { Button, Container, Grid, Typography } from '@mui/material'
+import LinkButton from '@/components/common/LinkButton'
+import { WALLET_LINK } from '@/config/constants'
+import type { BaseBlock } from '@/components/Home/types'
+import layoutCss from '@/components/common/styles.module.css'
+import css from './styles.module.css'
+
+const CardContent = ({ caption, title, text, link }: Partial<BaseBlock>) => (
+  <div className={css.cardContent}>
+    <Typography variant="caption" className={css.caption}>
+      {caption}
+    </Typography>
+
+    <Typography variant="h4" className={css.title}>
+      {title}
+    </Typography>
+
+    <Typography color="primary.light">{text}</Typography>
+
+    {link ? (
+      <LinkButton underline={false} href={link.href}>
+        {link.title}
+      </LinkButton>
+    ) : undefined}
+  </div>
+)
+
+type FeatureCardsProps = Omit<BaseBlock, 'items'> & {
+  items: Array<Partial<BaseBlock> & { isNew: boolean; fullWidth: boolean }>
+}
+
+const FeatureCards = ({ title, text, items = [] }: FeatureCardsProps) => {
+  // extract the last item from the array
+  const [lastItem, ...restItems] = [...items].reverse()
+
+  return (
+    <Container className={layoutCss.containerMedium}>
+      <Grid container justifyContent="space-between" columnSpacing="30px" rowGap="30px">
+        <Grid item xs={12} md={7}>
+          <Typography variant="h2">{title}</Typography>
+        </Grid>
+
+        <Grid item xs={12} md={5} alignContent="flex-end">
+          <Typography variant="h5">{text}</Typography>
+        </Grid>
+      </Grid>
+
+      <Grid container mt={{ xs: '50px', md: '100px' }} columnSpacing="30px" rowGap="30px">
+        {restItems.reverse().map((item, index) => {
```

Comment:
You can save the double reverse by using `slice`:
```suggestion
  const lastItem = items[items.length - 1]
  const restItems = items.slice(0, -1)

  return (
    <Container className={layoutCss.containerMedium}>
      <Grid container justifyContent="space-between" columnSpacing="30px" rowGap="30px">
        <Grid item xs={12} md={7}>
          <Typography variant="h2">{title}</Typography>
        </Grid>

        <Grid item xs={12} md={5} alignContent="flex-end">
          <Typography variant="h5">{text}</Typography>
        </Grid>
      </Grid>

      <Grid container mt={{ xs: '50px', md: '100px' }} columnSpacing="30px" rowGap="30px">
        {restItems.map((item, index) => {
```

Reply:
Good suggestion!